README.md

```diff
@@ -152,29 +154,28 @@ A tutorial using this method will soon be available in Grafana Cloud's docs.
 The following instructions will help you go from metrics to logs to traces.
 
 1. Open the TNS dashboard.
-2. Zoom in on a section of a panel with failed requests.
-3. From the panel drop-down, click Explore.
-4. In the Explore view, go to the data source drop-down and select Loki.
-5. Click to expand a logline with a `traceID` field.
-6. Click the Tempo button next to the `traceID` field to view the trace.
+1. Zoom in on a section of a panel with failed requests.
+1. From the panel drop-down, click Explore.
+1. In the Explore view, go to the data source drop-down and select Loki.
+1. Click to expand a logline with a `traceID` field.
+1. Click the Tempo button next to the `traceID` field to view the trace.
 
 ## Explore metrics to traces to logs
 
 The following instructions will help you go from metrics to logs to traces.
 
 1. In Grafana, go to the Explore view.
-2. From the data source drop-down, select "Prometheus-Exemplars".
-3. Run the following query: `histogram_quantile(.99, sum(rate(tns_request_duration_seconds_bucket{}[1m])) by (le))`
-4. Click on a data point to see the exemplar data as a tooltip. (If no data points appear, make sure exemplars are enabled in the query options.)
-5. Click on the log icon on a span line to view the log details.
+1. Run the following query: `histogram_quantile(.99, sum(rate(tns_request_duration_seconds_bucket{}[1m])) by (le))`
```

Comment:
It may still be helpful to make sure the correct data source is selected ("Mimir")